scaling/plugins/torque.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+import shlex
+import logging
+
+log = logging.getLogger(__name__)
+
+# get nodes requested from pending jobs
+def pending(instance_type):
+    command = "qstat -a"
+    return run_command(command, ['C', 'Q'])
+
+# get nodes requested from pending jobs
+def running(instance_type):
+    command = "qstat -a"
+    return run_command(command, ['R'])
+
+def run_command(command, status):
+    args = shlex.split(command)
+    pending = 0
+    try:
+       _output = subprocess.check_output(args)
+       output = _output.split("\n")[5:]
+       for line in output:
+          line_arr = line.split()
+          print(line_arr)
+          if len(line_arr) >= 10 and line_arr[9] in status:
+             pending += int(line_arr[5])
```

Comment:
the variable should be renamed to something like 'njobs'  since they are not necessarily pending jobs

Reply:
changed.